cylc/flow/option_parsers.py

```diff
@@ -335,11 +326,11 @@ class CylcOptionParser(OptionParser):
             help=(
                 'A more convenient alternative to --set for defining a list'
                 ' of strings. E.G.'
-                ' "-z FOO=a,b,c" is shorthand for'
-                ' "-s FOO=[\'a\',\'b\',\'c\']".'
+                ' "-z FOO=a,b,c" is shorthand for'
+                ' "-s FOO=[\'a\',\'b\',\'c\']".'
                 ' Commas can be present in values if quoted, e.g.'
-                ' "-z FOO=a,\'b,c\'" is shorthand for'
-                ' "-s FOO=[\'a\',\'b,c\']".'
+                ' "-z FOO=a,\'b,c\'" is shorthand for'
+                ' "-s FOO=[\'a\',\'b,c\']".'
```

Comment:
I think you haver replaced functional non-breaking spaces here. Please don't set up your editor to auto-convert.

Reply:
I did this manually assuming they were mistaken. But now I see why they were deliberate